lisa-utils/src/main/scala/lisa/utils/K.scala

```diff
@@ -0,0 +1,15 @@
+package lisa.utils
+
+object K {
```

Comment:
I agree that it's nice to be able to use it as K, but it's probably better to name it `Kernel` and alias it after, even a `val K = Kernel` right next to it is fine

Reply:
This is already supposed to be the alias :)